pymc_marketing/mmm/base.py

```diff
@@ -664,590 +636,7 @@ def compute_channel_contribution_original_scale(self) -> DataArray:
-        -------
-        plt.Figure
-            A matplotlib Figure object with the direct contribution curves.
-        """
-        channels_to_plot = self.channel_columns if channels is None else channels
-
-        if not all(channel in self.channel_columns for channel in channels_to_plot):
-            unknown_channels = set(channels_to_plot) - set(self.channel_columns)
-            raise ValueError(
-                f"The provided channels must be a subset of the available channels. Got {unknown_channels}"
-            )
-
-        if len(channels_to_plot) != len(set(channels_to_plot)):
-            raise ValueError("The provided channels must be unique.")
-
-        channel_contributions = self.compute_channel_contribution_original_scale().mean(
-            ["chain", "draw"]
-        )
-
-        if same_axes:
-            nrows = 1
-            figsize = (12, 4)
-
-            def label_func(channel):
-                return f"{channel} Data Points"
-
-            def legend_title_func(channel):
-                return "Legend"
-
-        else:
-            nrows = len(channels_to_plot)
-            figsize = (12, 4 * len(channels_to_plot))
-
-            def label_func(channel):
-                return "Data Points"
-
-            def legend_title_func(channel):
-                return f"{channel} Legend"
-
-        fig, axes = plt.subplots(
-            nrows=nrows,
-            ncols=1,
-            sharex=False,
-            sharey=False,
-            figsize=figsize,
-            layout="constrained",
-        )
-
-        axes_channels = (
-            zip(repeat(axes), channels_to_plot)
-            if same_axes
-            else zip(np.ravel(axes), channels_to_plot, strict=False)
-        )
-
-        for i, (ax, channel) in enumerate(axes_channels):
-            if self.X is not None:
-                x = self.X[channels_to_plot].to_numpy()[:, i]
-                y = channel_contributions.sel(channel=channel).to_numpy()
-
-                label = label_func(channel)
-                ax.scatter(x, y, label=label, color=f"C{i}")
-
-                if show_fit:
-                    label = f"{channel} Fit Curve" if same_axes else "Fit Curve"
-                    self._plot_response_curve_fit(
-                        x=x,
-                        ax=ax,
-                        channel=channel,
-                        color_index=i,
-                        xlim_max=xlim_max,
-                        method=method,
-                        label=label,
-                    )
-
-                title = legend_title_func(channel)
-                ax.legend(
-                    loc="upper left",
-                    facecolor="white",
-                    title=title,
-                    fontsize="small",
-                )
-
-                ax.set(xlabel="Spent", ylabel="Contribution")
-
-        fig.suptitle("Direct response curves", fontsize=16)
-        return fig
-
-    def _get_distribution(self, dist: dict) -> Callable:
```

Comment:
Does this method get removed or not? It is hard to  tell in GitHub diff

Reply:
The only reason to move it was `mypy`.